k8s.gcr.io/images/k8s-staging-build-image/images.yaml

```diff
@@ -6,14 +6,16 @@
     "sha256:1c81250e8dc13a7b1c9fbd7a7f47843e1bc7c66cbf308f315772b88673a82bf2": ["buster-v1.6.0"]
     "sha256:1e76a235c477dfe46d707d2be80a835b44cdcf6f35675fb2189c7a28b6d09878": ["buster-v1.9.0"]
     "sha256:22666783ee41fa619ad4d7ea40800bb40901d2e27d60c0ca3339a5851374763e": ["buster-v1.8.0"]
+    "sha256:248087165d8f4d901e27a799f10fe4b7b4458469191120385317ad32aa5e3382": ["buster-v1.10.0"]
     "sha256:2f37650255a457d0fc3a5c685ac1eecb5e0f16c63b58b82e52a585fff0aabff6": ["buster-v1.7.2"]
     "sha256:36652ef8e4dd6715de02e9b68e5c122ed8ee06c75f83f5c574b97301e794c3fb": ["buster-v1.4.0"]
-    "sha256:37cfe133a6ff3fc3aa4aa5ab9fda127861902940b8e8078fff7191c7a81be8d8": ["buster-v1.1.3","v2.1.3"]
+    "sha256:37cfe133a6ff3fc3aa4aa5ab9fda127861902940b8e8078fff7191c7a81be8d8": ["buster-v1.1.3", "v2.1.3"]
```

Comment:
lol. `cip-mm` adds space for readability, while `kpromo pr` [strips them](https://github.com/kubernetes/k8s.io/pull/3092/files#r752684922).
I can fix that up later.